release-notes/2.1/Preview/2.1.0-preview2-known-issues.md

```diff
@@ -26,4 +26,24 @@ We have temporarily removed the Preview 2 installers from the Linux package feed
 
 ```bash
 sudo apt install dotnet-host=2.0.6-1
-```
\ No newline at end of file
+```
+
+## SocketsHttpHandler hangs on single core machines
+
+There is an issue with the new default HttpClientHandler that will cause hangs on single core machines when multiple connections are opened simultaneously. [corefx/issues/28979](https://github.com/dotnet/corefx/issues/28979). This issue has since been fixed, but is present in the Preview 2 build.
+
+**Workaround** - Disable SocketsHttpHandler as the default, and fall back to the platform handler.
```

Comment:
Let's reword it to "Use 2+ core machines, or use older HttpClientHandler (as described in [Preview 2 blog post](https://blogs.msdn.microsoft.com/dotnet/2018/04/11/announcing-net-core-2-1-preview-2/))"

Reply:
Updated this in the latest commit.